### DISCUSSION TITLE
Who wants to be hired (Oct 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#41662] [Ideas]
# What is this?
A little late but better late than never! This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #41663.

*If you're looking for last month's discussion, see #39274. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!*

**Comment by ThatGuySam** [comment]
Location: Tulsa, OK
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, Tailwind, Node.js, Serverless Functions, Vite, React, Vue, Next.js, Nuxt.js, Astro, Netlify, Vercel
Résumé/CV: https://docs.google.com/document/d/1cLxJU61AzpHIeC7yD-VB-I5q5T1CyI2k03qj9Xbkvp8/edit#
Website: https://github.com/ThatGuySam
Email: hnhiringoct22@scc.mozmail.com
LinkedIn: https://www.linkedin.com/in/sam-carlton-99884629/


Hey, I'm Sam, and I'm a Senior Front End Engineer with 10+ Years of Experience. 

I love long walks on the beach and taking on impossible-to-solve Front End Engineering challenges. 

For client projects, I recently worked on a WordPress site for a Manhattan Synagogue that involved a heavy rewrite in Tailwind. 

Another recent client was a React PWA built with Next.js, where I also got to implement Natural Language Processing via JavaScript to make event dates smarter. 

I also built and currently maintain the largest GitHub community for Apple Silicon Software with 94 open-source contributors.
https://github.com/ThatGuySam/doesitarm


In my spare time, I built an interface for displaying the order of Marvel Movies built on TypeScript + Astro, which allows you to build with React, Vue, Svelte, or any Vite compatibles JS Framework: 
https://github.com/ThatGuySam/marvelorder

**Reply by kitinmayo** [reply]
Hello, I am Chritine Guiuan,  sourcing specialist at Stoke Talents. I am impressed with your Node background and would like to invite you to apply for our Remote Team Lead for Node. It is a 10-month freelance contract with a negotiable salary. Please let me know if you're interested. Thank you.

**Comment by zhyd1997** [comment]
Hi 👋 there

I am Yadong Zhang from China and am a frontend developer, also a freelancer.

Currently just looking for a **full-time** job, let me introduce myself more:

After graduated in July 2019 with Bachelor's degree (no CS background), I started my CS journey for a year, and then got my first job at **Compass**: https://github.com/UrbanCompass in March 2021, but just working for a few months, that was a good experience in the listed company.

In the second half of the year, of course, I applied many job opportunities (Including ByteDance and RedHat) but unfortunately, all of them failed, then I found a part-time job at **Stargately**: https://github.com/stargately/ (web-boommo app, which is private in GitHub), I learned that every developer should have his/her own product so I launched my own side project called **SoftMaple**: https://github.com/softmaple/softmaple, which is a paper typesetting editor for professors and students. The project is promoted by Github in Dec 2021: https://github.blog/2022-01-21-release-radar-dec-2021/

And in 2022, I became a freelancer at Web3 field:

Contributed to

- **Superfluid** ecosystem: (bug fixes and fixtures)

1. Ricochet-Exchange: https://github.com/Ricochet-Exchange/ricochet-frontend

2. Geo-Web: https://github.com/Geo-Web-Project/cadastre

- **Synthetix** ecosystem: (bug fixes and features)

  - Kwenta: https://github.com/Kwenta/kwenta

- **Uniswap** (small bug fixes)

  - https://github.com/Uniswap/interface

Also traditional web2 open source project:

- **Storybook** (small bug fixes)

1. storybook: https://github.com/storybookjs/storybook

2. design-system: https://github.com/storybookjs/design-system/

3. eslint-plugin-storybook: https://github.com/storybookjs/eslint-plugin-storybook

- **ESLint** ecosystem (small bug fixes)

  - typescript-eslint: https://github.com/typescript-eslint/typescript-eslint

Location: **China**
Remote: **Yes**
Willing to relocate: **Yes**
Technologies: **React**, **Nextjs**, **Emotionjs**, **TypeScript**
Resume/CV: https://www.linkedin.com/in/yadong-zhang-48a474154/
Website: https://github.com/zhyd1997
Email: zhyd007@gmail.com

**Comment by kroucher** [comment]
Location: **Queensland, Australia**
Remote: **Yes**
Willing to relocate: **Possibly**
Technologies: **Next.JS, ES6+ JavaScript, TypeScript, Node.js, React.JS, REST API/tRPC, SQL/NoSQL, Prisma, Three.js, Azure/AWS**
Resume/CV: https://www.linkedin.com/in/daniel-deveney-98b94b249/
Website: [Personal Portfolio](https://portfolio.beanlea.com/)
Email: kroucher.1019@gmail.com

Extremely fast learner, after a decade in the senior system administration and project delivery space, I fell in love with code through automation with PowerShell, taught myself JavaScript, now I’m building cool stuff and enjoying every minute.

Confident and ready to start contributing great things to a diverse team, looking for anything from internships to mid-level engineer roles, backend, frontend or full stack.

**Comment by js-matt** [comment]
Location: Chicago, Il
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Tailwind, Node.js, Serverless Functions, Vite, Vue, Next.js, Nuxt.js, Netlify, Vercel, AWS lambda, AWS S3, Styled Component, SCSS
Résumé/CV: https://drive.google.com/file/d/1xewGxISgNQBg9qKve_rFDZKMglA2lAnY/view?usp=sharing
Website: https://github.com/js-matt
Email: [imatthewnguyen@gmail.com](mailto:imatthewnguyen@gmail.com)
LinkedIn: https://www.linkedin.com/in/matthew-nguyen-dev/

I am a team-oriented professional with over 6 years of experience in a high-volume distribution center. My natural love of problem solving and continual learning have aided me well in becoming a full-stack web developer, and I am eager to ﬁnd a team in which I can add value and with whom I can continue my growth as a software engineer

**Reply by kitinmayo** [reply]
Hello, I am Chritine Guiuan,  sourcing specialist at Stoke Talents. I am impressed with your Node background and would like to invite you to apply for our Remote Team Lead for Node. It is a 10-month freelance contract with a negotiable salary. Please let me know if you're interested. Thank you.

**Comment by SuperButterfly** [comment]
Location: Edmonton, Canada
Remote: Yes
Willing to relocate: No
Technologies: React.js, Blockchain ,Node.js, Tailwind CSS, TypeScript
Resume:[ERic lee (Blockchain& React).pdf](https://github.com/vercel/next.js/files/9848002/ERic.lee.Blockchain.React.pdf)
Website: https://github.com/liger4989
Email: [ericlee4989@gmail.com)

As a passionate and innovative full stack developer with 6+ years experience, I developed and
innovated so many precious things.
At First, I was trying to master React.js,Node.js and those days, I developed many successful
websites that had various purposes(E-Commerce, AWS, Voting, Advertisement...).
And then I jumped in to Blockchain world.
In the great blockchain world, I learned about coins and tokens such as BTC,SOL,ETH and also
got knowledge of so many key-skills such as Solidity, Web3.js,NFT marketplace.
Now, I am always ready for challenges and obstacles to create the next generation and
innovation in a growing world.

**Comment by vincentabolarin** [comment]
**Location:** Lagos, Nigeria

**Remote:** Yes

**Willing to relocate:** Yes

**Technologies:** HTML, CSS, JavaScript, React, Next.js, Angular, CSS Modules, Styled-Components, Git, Netlify, Vercel

**Resume:** https://drive.google.com/file/d/1chzj3oufxMRPJVzjSYqGP-PTmu7V3aC4/view?usp=drivesdk

**Website:** https://vincentabolarin.dev

**LinkedIn:** https://www.linkedin.com/in/vincentabolarin

**Email** vincentabolarin@gmail.com

Hello,

I am Vincent, a Frontend Engineer with 3 years of experience building applications using various technologies.

In addition to Core Development, I also build business applications using the Microsoft Power Platform.

**Comment by IsrafilliRamin** [comment]
Location: Azerbaijan Baku

Remote: Yes

Willing to relocate: No

Technologies: HTML, CSS, JavaScript, React,Redux, Bootstrap, CSS Modules, Styled-Components, Git,  Vercel , REst API, , Tailwind Css , Sass/Scss , Jquery, Typescript , Ant design 

Resume: https://www.canva.com/design/DAFHK6iz5JE/73fURJPeXscANRVuZKSKsg/edit?layoutQuery=resume#

Website: https://github.com/IsrafilliRamin

LinkedIn: https://www.linkedin.com/in/ramin-israfilli-014420246/

Email (raminisrafilli@gmail.com)

Hello,

I am  Frontend React Developer with1 years of experience building applications using various technologies.

**Comment by SuperFSDev** [comment]
I'm a full-stack developer with 4+ years of experience in web development. Spearheaded successful build of the website in MEAN stack, Ruby on Rails, and Django as a backend framework in large projects.

Location: Colombia
Remote: Yes
Willing to relocate: No
Technologies: React Native, Ruby on Rails, JavaScript, TypeScript, ReactJS, NodeJS, NextJS, Express, HTML, CSS, Python, Java, D3, Swift, Git, AWS, MongoDB, Redux, Bootstrap, Material UI, SQL

Resume/CV: https://github.com/vercel/next.js/files/9596276/Jefrey.pdf
Email: (beyondlimitation9@gmail.com)

**Comment by mattspaulding** [comment]
Hi I'm Matt! I have a Masters in Computer Engineering and over 13 years of professional experience. My last 3 years I have been a technical Engineering Manager. I'm seeking a hands-on technical leadership role.

Location: Tampa FL USA Eastern Time
Remote: Yes
Willing to relocate: No
Technologies: Touched all sorts of languages, but looking for MERN stack or similar. And Nextjs of course ;)
Resume/CV: https://www.mattspaulding.org/resume.pdf
Website: https://www.mattspaulding.org/
LinkedIn: https://www.linkedin.com/in/spauldingmatthew/
Comp: $220k

**Comment by JosephAni** [comment]
Hello, am Joseph Ani a frontend software engineer currently teaching @ LaunchCode. I specialize in building, designing, and strategizing software digital experiences. Currently, am focused on innovative responsive full-stack web applications.
 
Location: Philadelphia, PA, USA.
Remote: Yes
Willing to relocate: No
Technologies: Next js, React, Node js, Typescript, Redux, Mongo, SQL, Solidity
Resume/CV: https://drive.google.com/file/d/1BUKE6B6ABZVhYLgledYdp42_JQOIm72v/view?usp=share_link
Website: https://my-portfolio-josephani.vercel.app/
Email: joeoani@gmail.com

**Reply by kitinmayo** [reply]
Hello, I am Chritine Guiuan,  sourcing specialist at Stoke Talents. I am impressed with your Node background and would like to invite you to apply for our Remote Team Lead for Node. It is a 10-month freelance contract with a negotiable salary. Please let me know if you're interested. Thank you.

**Reply by JosephAni** [reply]
Hello Kitinmayo, I am interested in learning more about the job, and I would like to apply for the position of remote team lead. Please let me know if there is anything else that I can provide.
Thanks

**Comment by davidnoelopez** [comment]
Hi, I'm David Lopez a fullstack developer looking for new opportunities to grow. I been working in software for a while (10+ years) managing business processes, optimizing and automating everyday tasks. I've been working with React and Next.js for the past year and I'm looking to work and learn more with the tech stack.

Location: Monterrey, Mexico.
Remote: Yes
Willing to relocate: No
Technologies: Next.js, Node, React, PHP, Mongo.
Resume/CV: shorturl.at/DKX89
Website: https://github.com/davidnoelopez
Email: davidnoelopez@gmail.com

**Reply by kitinmayo** [reply]
Hello, I am Chritine Guiuan,  sourcing specialist at Stoke Talents. I am impressed with your Node background and would like to invite you to apply for our Remote Team Lead for Node. It is a 10-month freelance contract with a negotiable salary. Please let me know if you're interested. Thank you.

**Comment by diegoaguilar** [comment]
Timezone: CDT
Looking for: Contractor work
Remote: Yes
Languages: Typescript, Python
AWS: CodePipeline, Lambda, CodeBuild, IAM, RDS, Fargate, EventBridge, SQS, SNS, etc
IaC: Terraform, Pulumi, CDK
Experience: Contentful, Stripe, Auth0, Twillio, databases as service
Web: Next & React

Email: [diegoaguilaraguilar@gmail.com](mailto:diegoaguilaraguilar@gmail.com])

**Comment by D3troit98** [comment]
Location: Lagos, Nigeria
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Tailwind, Node.js, Next.js, Netlify, Vercel, C++, Styled Component, SCSS
Resume: https://drive.google.com/file/d/1CRQ0BRIlqrjbglFYMEwzJzRt9ac9YUiT/view?usp=drivesdk
Website: https://portfolio-1-theta.vercel.app/
Email: Duruakuebuka@gmail.com
LinkedIn: https://www.linkedin.com/in/duruaku-ebuka-6b6540137

I am a junior developer with almost two years of web development and I'm looking for opportunities to showcase my skills

**Comment by AndrewNow** [comment]
Hey everyone! UX/UI Designer and front-end dev here with 5-6 years of professional freelance experience. I typically run projects from A-Z (brand planning & strategy -> low fidelity/high fidelity mockups -> web development) but also have experience working in bigger teams, helping me understand how every piece of the puzzle fits together when working on projects of any scale.

**A few examples of recent work**: [Eight Branches](https://www.eightbranches.ca/), [Revue Moebius](https://www.revuemoebius.com/)


Timezone: **EST** (Montreal, Canada)
Looking for: **Contractor work**
Remote: **Yes**
Tech: Mainly NextJs/Framer Motion/Sanity CMS/Netlify CMS/Stripe/Shopify

Email: [andrew.p.nowacki@gmail.com](mailto:andrew.p.nowacki@gmail.com)

I pour my heart and soul into every project I take on. If you have any projects you'd like to work on together, please don't hesitate to hit me up! Let's make something special together :)

**Comment by spencerpauly** [comment]
Hiya there👏. I'm currently looking for a full time frontend/full stack SWE role based in the U.S. A little about me:

🚀 Building and growing a paid B2B SaaS tool around Notion  to **$4000/year** revenue (https://engine.so/)
🚀 Building and growing a cross-platform app to **12,000+ users** using React, Ionic, cross-platform (https://skiwise-app.com/). 
⭐ Working as a L4 Software Engineer at Target  

**Looking for**: Full-time
**Location**: USA (Minneapolis, MN)
**Remote**: Yes
**Willing to relocate**: No
**Technologies**: Deep experience with TypeScript / JavaScriptReact, Next.js,  React, Redux, TailwindCSS, Node.js, Serverless, GraphQL, AWS, Docker, Github Actions, Postgres, SQL
**Resume/CV**: https://drive.google.com/file/d/1ZYBAPsxmkENo6mE3xk1M57ADB1--tcl0/view?usp=sharing
**Website**: https://spencerpauly.com/tech
**Email**: spencerpauly@gmail.com

I'd love to chat sometime, feel free to reach out if you have an opportunity.

**Reply by kitinmayo** [reply]
Hello, I am Chritine Guiuan,  sourcing specialist at Stoke Talents. I am impressed with your Node background and would like to invite you to apply for our Remote Team Lead for Node. It is a 10-month freelance contract with a negotiable salary. Please let me know if you're interested. Thank you.

**Comment by tajnur007** [comment]
Location: Dhaka, Bangladesh
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next.js, TypeScript, Redux, React-Query, Tailwind, Bootstrap, Material UI, Node.js, Express.js, MongoDB, Firebase.
Resume/CV: https://drive.google.com/file/d/1RHw0wR3cFXEAcwDMOD5EG3iQ0sQs27J4/view
Website: https://github.com/tajnur007
Email: tajnur007@gmail.com
LinkedIn: https://www.linkedin.com/in/tajnur/

**Comment by mastercodercat** [comment]
As a senior frontend developer specializing in React and Next.js with TypeScript, Shane(me) is now looking for new positions and growth opportunities. With over 9 years of experience in front-end development, I've worked as a frontend-heavy full stack developer, and participated in web/mobile projects with modern technologies and libraries, mainly with React, React Native, and Next.js.
I prefer to work on a growing team and startups with great teammates and prefer a fully-remote work environment.
I am available for full-time positions as well as part-time or contract positions in global teams.

Location: Canada
Remote: Yes
Willing to relocate: No
Technologies: **Next.js**, **React**, **React Native**, **Ionic**, **TypeScript**, **GraphQL**, **Cypress**, **RTL**, **Tailwind**, **Node.js**, **Express.js**, **Nest.js**, **Prisma**
Resume: [My Resume](https://drive.google.com/file/d/1fw3_udibFbQyhubnga_oqeejMLCgUA3n/view?usp=sharing)
Website: [Portfolio](https://shanelewis.vercel.app)
Email: shanelewix@gmail.com

**Comment by Devang47** [comment]
Location: India
Remote: Yes
Willing to relocate: No
Technologies: Typescript, React js,  Svelte, Tailwind, Golang, MongoDB, Next js, Node, Express, Prismic
Resume/CV: https://saklani.dev/resume.pdf
Website: https://saklani.dev/
Email: devangsaklani@gmail.com

Hello, I’m Devang Saklani, I have 1 year of professional experience as a Web developer. I specialize in technologies like Typescript, Sveltekit, Figma, and Nextjs. I'm now looking for new positions and growth opportunities. I've worked with design agencies like Monogram and IDS, at these agencies I have contributed to creating beautiful and performant Jamstack applications using Nextjs, Sveltekit, and CMS like Sanity, and Prismic.
I have also designed many personal websites and side projects like my portfolio (saklani.dev) and rooms.saklani.dev.

**Comment by vigneshshiv** [comment]
I have been a Full Stack Engineer for the last 8 years and I specialized in Microservices Architecture and application development. I've worked in Banking, Airline and Automobile industry. 

So far I have worked on various technical challenging projects from the ground up and I do open source contributions and self initiated a project called Online Classroom.

Location: India
Remote: Yes
Willing to relocate: No
Technologies: Angular, React, Next.js, TypeScript, JavaScript, React-Query, Tailwind CSS, Prisma, Java, Spring Boot, Oracle, MSSQL, Azure
Resume/CV: https://github.com/vigneshshiv/resume
Website: https://vigneshshiv.github.io/
Email: vichuviky@gmail.com

**Comment by stefkudla** [comment]
Professional Software Developer and Technical Content Writer. A motivated and efficient creator of high-quality Software products and a great team player with phenomenal time management skills.

Making an impact at start-ups by architecting a fluent developer experience, building product-based software templates, and writing SEO-rich technical articles, pages, and documentation. An eye for detail and a drive to educate your future users about your product or service.

Location: Washington D.C. Area
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Next.js, Astro, Tailwind CSS, Node.js, Cosmicjs, Hygraph (GraphCMS), Framer Motion, HTML, CSS, Firebase, Figma, Git, Vercel, Netlify, Serverless
Resume/CV: [Stefan Kudla's Resume](https://stefankudla.com/Stefan_Kudla_Resume.pdf)
Website: https://stefankudla.com/
Email: [stefan@stefankudla.com](mailto:stefan@stefankudla.com)

**Comment by FLCN-16** [comment]
Professional Full Stack Developer with over 6 years of experience on live projects. Known for High quality and optimised solutions provider.

Location: Punjab, India
Remote: Yes
Willing to relocate: Yes
Technologies: React, Next, Redux, Redux Saga, Redux Thunk, Vue, Nuxt, Prisma, Graphql, TypeScript, Php, Laravel
Resume/CV: https://zety.com/profile/flcn16/415
Email: work@thefalcon.dev

**Comment by Anishpras** [comment]
Hi,
My name is Anish Prashun. I have 4+ years of development with React.js and 2+ years with Next.js. I have worked on more than 25+ Projects as a freelancer. I am looking for a Fullstack / Frontend position. Currently, I am working as a Frontend Developer on a contract basis at The Volte, an Australia-based e-commerce company, and at Xamtac Consulting (a USA-based Marketing Consultancy Services company) as a lead Software Developer and leading the development team. 

Location: Bihar, India

Remote: Yes

Willing to relocate: Yes

Technologies: TypeScript, JavaScript, React, NextJS, tRPC, Prisma, GraphQL, Postman, PostgreSQL, SQL, MongoDB, NodeJS, TailwindCSS, SCSS, Redux / Recoil / Zustand, Docker.

Resume / CV : [anish-resume.pdf](https://github.com/vercel/next.js/files/9880631/ANISH_RESUME_28_09_2022.1.pdf)

Website: https://www.anishprashun.me/

LinkedIn: https://www.linkedin.com/in/anishpras118/

Email: [anishprashun118@gmail.com](mailto:anishprashun118@gmail.com)

**Comment by horatiuvlad** [comment]
Remote: Yes 
Technologies: ReactJS, React Native, TypeScript, NextJS, Expo, NodeJS, Tailwind CSS, Prisma ORM, Hasura, PostgreSQL, MongoDB, GraphQL, Apollo, Docker, Kubernetes, Vercel, Firebase, Sentry
 Resume/CV: https://www.linkedin.com/in/horatiu-vlad/ 
Website: https://foreverworks.io/
Email: [horatiu@foreverworks.io](mailto:horatiu@foreverworks.io)

Hi!

I’m Horatiu, a cloud architect and founder of Forever Works, a development agency offering hybrid, web and mobile applications, alongside legacy to cloud services.

My 10+ years of experience is backed by a skilled team of contracted developers and engineers who are passionate about helping businesses succeed through cutting-edge technology. We have a proven track record of delivering high-quality projects on time and within budget.

So, what makes us different from other development agencies? Preparation is key, we devise comprehensive blueprints for our customers’ platforms, detailing a no-hassle path to success. This allows our developers to provide premium deliverables from flawless interactions to clean user interfaces. With our meticulous development process, our clients experience significantly lower issues, resulting in a better return on their investment.

If you are interested in learning more about our services, please don't hesitate to schedule a call with me. I would be happy to answer any questions you have and provide you with a no obligation quote. You can also find out more about our services on our website: https://foreverworks.io/

Thank you for your time,
Horatiu

**Comment by SuperStar518** [comment]
Location: Hongkong, OK
Remote: Yes
Willing to relocate: Yes
Technologies: TypeScript, Tailwind, Node.js,  React, Vue.js, Next.js, Nuxt.js, PHP, Laravel
Resume/CV: https://drive.google.com/file/d/1ZfzD2j4uLlPV9Q-SrztwUvZ_0bGFpEvb/view?usp=sharing
Website: https://github.com/SuperStar518
Email: robbie626827@gmail.com

Hello, Sir.
I am Ali David.
Going through your project description I found it right fit for us.
I can do such app from your posting and I will send you awesome result within your deadline if you hire me.
I have 5+ years’ experience working with JavaScript and PHP frameworks, preferably React/Redux.
I have depth understanding of JavaScript , the DOM Elements and relevant concepts.
My primary focus will be on developing user interface components and implementing them following well-known React.js workflows (such as Flux or Redux and GraphQL or ApolloClient).
I will ensure that these components and the overall application are robust and easy to maintain.
I will try to commit to collaborative problem solving, guarantee sophisticated design, and quality product.
High coding quality guaranteed~
Also I am good at proficiency with browser debugging and performance testing and have good experience using Git and MVP frameworks including Jira.
I would like to inform you also that I am highly skilled with Node.js, Firebase, MongoDB, WebPack, ES6, API Integration, as I have a handsome experience over it.
I firmly believe that these qualities make me different from the rest of the contestants and also are exactly fit to the skills of your requirements.
I am working on full-time job with freelancer and can work any time you want so feel free to keep in touch with you anywhere and whenever you want.
I would like to discuss more details in chat or interview so we can get clear idea with each other.
Best regards.

**Comment by clevermacaw** [comment]
Hi, my name is Alex Lim and have a strong background in the frontend development.

Location: Santa Clara, CA
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Next, Vue, TailwindCSS, Material UI, Storybook
Resume/CV: https://alex-lim.vercel.app/assets/resume.pdf
Website: https://alex-lim.vercel.app
Email: [cleverbluemaccaw@gmail.com](mailto:cleverbluemaccaw@gmail.com)

**Comment by Mugisha-isaac** [comment]
> # What is this?
> A little late but better late than never! This discussion is for Next.js developers looking for work.
> 
> # Candidates
> Share your information if you are looking for work. Please use this format:
> 
> Location: Remote: Willing to relocate: Technologies: Resume/CV: Website: Email:
> 
> Good luck with interviewing!
> 
> If you got hired through this job board, please reach out to me ([hassan@vercel.com](mailto:hassan@vercel.com)) so I can send you some swag :)
> 
> If you're a company looking to post a role, use #41663.
> 
> _If you're looking for last month's discussion, see #39274. We've had another [_success story_](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!_



> # What is this?
> A little late but better late than never! This discussion is for Next.js developers looking for work.
> 
> # Candidates
> Share your information if you are looking for work. Please use this format:
> 
> Location: Remote: Willing to relocate: Technologies: Resume/CV: Website: Email:
> 
> Good luck with interviewing!
> 
> If you got hired through this job board, please reach out to me ([hassan@vercel.com](mailto:hassan@vercel.com)) so I can send you some swag :)
> 
> If you're a company looking to post a role, use #41663.
> 
> _If you're looking for last month's discussion, see #39274. We've had another [_success story_](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!_

Hi my name is Isaac Mugisha I am a full-stack developer
Location: Rwanda
Remote: Yes
Willing to relocate: No
Technologies: Nodejs, Express, Nestjs, Nextjs, Typescript, Reactjs, Remix, 
Website:
Email: mugishaisaac0508@gmail.com

Resume/CV:
[MUGISHA ISAAC.pdf](https://github.com/vercel/next.js/files/10064752/MUGISHA.ISAAC.pdf)

**Comment by clevermacaw** [comment]
Hello Chritine,

I am interested in it. How can I apply?

Best,
Alex

**Reply by kitinmayo** [reply]
Where can I reach you so we could discuss about this further? Do you have LinkedIn Account?

**Comment by clevermacaw** [comment]
Sure, https://linkedin.com/in/alex-lim-38478b225

**Reply by kitinmayo** [reply]
Messaged you. Please check your inbox.

**Comment by SuperButterfly** [comment]
hi how are you?

**Comment by erklyn** [comment]
Location: Sakarya, Turkey
Remote: Yes
Willing to relocate: Yes
Technologies: React.js, Typescript-Javascript, Next.js, Kotlin, GOlang, Astro
Resume/CV: [LinkedIn/Resume](https://www.linkedin.com/in/utku-enes-g%C3%BCrsel-98041517a/)
Website: [Blog](https://www.utkuenes.space/)
Email: [uenesgursel@gmail.com](mailto:uenesgursel@gmail.com)

Dropped out of Computer Engineering to join a Trailer Company on the 3rd quarter of 2021, built a CRM web application from ground-up using react.js on the frontend, express.js on the backend then deployed it to their physical servers back then on Ubuntu 18.04 with NginX. Moved a couple of companies worked remotely for all of them currently working as a Full Stack Developer for a British Company working with React w/Typescript on the front-end, Kotlin w/out a framework on the backend, Mongo as the Database of the choice. 

I am looking for an environment where I'll be able to try out new things while maintaining, delivering features, honestly I am looking to impact lives with my applications for the best.

I am willing to relocate.
